metadata.rb

```diff
@@ -21,8 +21,8 @@
 
 depends 'apache2'
 depends 'ark'
-depends 'database', '~> 2.3'
-depends 'mysql', '~> 5.0'
+depends 'database'
+depends 'mysql', '~> 6.0'
```

Comment:
Looks good to me, simply as it allows me to move forward with using this cookbook :)